kairon/api/app/routers/bot/bot.py

```
@@ -1700,3 +1703,91 @@
+    Drop an entire collection and its documents.
+    """
+    message, deleted_count = DataProcessor.delete_collection(
+        bot=current_user.get_bot(),
+        name=payload.collection_name
+    )
+    return Response(message=message, data={"deleted": deleted_count})
+
+
+@router.post("/collections/data", response_model=Response)
+async def list_collection_data(
+    payload: ListCollectionDataPayload = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)